src/cloud/proxy/nginx.conf

```diff
@@ -39,6 +39,9 @@ http {
     # Tune nginx keepalives to work with the GCP HTTP(S) Load Balancer:
     keepalive_timeout 650;
     keepalive_requests 10000;
+  
+    # Increase default server names lenght to 128:
```

Comment:
nit: can you fix the typo lenght -> "length"? It's triggering one of our lint rules. Thanks!

Reply:
Fixed it. And looks like your Jenkins build also passed